syncserver/src/web/transaction.rs

```diff
@@ -197,11 +195,10 @@ impl DbTransactionPool {
 }
 
 impl FromRequest for DbTransactionPool {
-    type Error = Error;
+    type Error = actix_web::Error;
```

Comment:
This was another change that felt kind of magical to me and I'd love some double-checking on.  This used to return `actix_http::Error`, which I think may have been a type alias for the `actix_web` version, but I'm not sure.  The reason the change is necessary, is `actix_http::Error` no longer implements `From<T: ResponseError>` after the upgrade.

Reply:
Yeah, this is pretty common for Actix, so I'm not surprised they moved `Error` again. Looks like the current version re-exports `actix_http::error::Error` which appears to be more about [HTTP processing errors](https://docs.rs/actix-http/3.5.1/src/actix_http/error.rs.html#81), rather than [`actix-web::error::Error`](https://docs.rs/actix-web/latest/actix_web/error/struct.Error.html), which appears to be very much about responses.